overlord/snapstate/storehelpers.go

```diff
@@ -198,6 +198,8 @@ func preUpdateInfo(st *state.State, snapst *SnapState, amend bool, userID int) (
 	return curInfo, user, nil
 }
 
+var ErrStoreUnresponsive = fmt.Errorf("the store is unable to process the request at this time (try again later)")
```

Comment:
ErrNoExpectedResult ?  (unresponsive sounds like a timeout kind of situation to me)

Reply:
ErrStoreDroppingRequests to be super specific?

Reply:
please no :) , that's too verbose and too specific for what is mostly a guess